lib/assessment.js

```diff
@@ -8,25 +8,80 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
 
+var min = function min(previousValue, currentValue, index, array) {
+  return previousValue < currentValue ? previousValue : currentValue;
 };
 
+console.log(reduce(numbers, min));
+console.log(numbers.reduce(function (previousValue, currentValue) {
+  return previousValue < currentValue ? previousValue : currentValue;
+}));
+
+
+};
+
+// I used Antony's Max Example of code and switched the operators.
+// I do not think I could logic out the whole thing as concisely as this.
+// I cannot write this code from scratch myself but I can read this and edit it
+
+
+
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
+
 var sum = function sum() {
 
+var sum = function sum(previousValue, currentValue, index, array) {
+  return previousValue + currentValue;
+};
+
+var numbers = [13, 26, 32, 5];
+
+console.log(reduce(numbers, sum));
+console.log(numbers.reduce(function (previousValue, currentValue) {
+  return previousValue + currentValue;
+}));
+
+
 };
 
+// Same case for solving here as above-
+// I heavily relied on Antony's code and can readi it pretty well
+// I Feel rushed so maybe I am not catching all of the edits but
+// I am concerned that I cannot construct this code myself,
```

Comment:
You're not expected to be able to write *his* code. You're expected to be able to write less elegant code that still works. This function works, but I'm not clear on what edits you've made to Antony's code.